src/test/test-audit-util.c

```diff
@@ -14,17 +14,15 @@ TEST(audit_loginuid_from_pid) {
 
         uid_t uid;
         r = audit_loginuid_from_pid(&self, &uid);
-        if (r != -ENODATA)
-                ASSERT_OK(r);
+        ASSERT_OK_OR(r, -ENODATA);
```

Comment:
When you touch this, please call the function in the macro to make the assertion message more informative:
```c
ASSERT_OK_OR(r = audit_loginuid_from_pid(&self, &uid), -ENODATA);
```